usecases/auth/authentication/oidc/middleware.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/weaviate/weaviate/usecases/config"
 )
 
-// Client handles the OIDC setup at startup and provides a middleware to tbe
+// Client handles the OIDC setup at startup and provides a middleware to the
```

Comment:
Judging by the rest of the comment, this looks like it should be `tbe` -> `be`

Reply:
Agree.